mappings/net/minecraft/util/math/Vec3d.mapping

```diff
@@ -55,3 +66,12 @@ CLASS net/minecraft/class_243 net/minecraft/util/math/Vec3d
 	METHOD method_24802 isInRange (Lnet/minecraft/class_2374;D)Z
 		ARG 1 pos
 		ARG 2 radius
+	METHOD method_24953 fromCenteredVec3i (Lnet/minecraft/class_2382;)Lnet/minecraft/class_243;
+		ARG 0 vec
+	METHOD method_24954 fromVec3i (Lnet/minecraft/class_2382;)Lnet/minecraft/class_243;
+		ARG 0 vec
+	METHOD method_24955 fromBottomCenteredVec3i (Lnet/minecraft/class_2382;)Lnet/minecraft/class_243;
+		ARG 0 vec
+	METHOD method_26410 fromCenteredVec3i (Lnet/minecraft/class_2382;D)Lnet/minecraft/class_243;
```

Comment:
ofBottomCentered

Reply:
Renamed to ofCentered, it's not necessarily the bottom, because the deltaY argument specifies how far up to make it